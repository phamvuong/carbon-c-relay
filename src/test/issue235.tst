collection.fo.bar.x.y.blakwklsa.skla
