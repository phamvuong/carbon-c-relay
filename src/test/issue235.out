listen
    linemode
        2003 proto tcp
        2003 proto udp
        /tmp/.s.carbon-c-relay.2003 proto unix
    ;

statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;

cluster aggrs
    fnv1a_ch replication 1
        127.0.0.1:2003=a
        127.0.0.1:2004=b
        127.0.0.1:2005=c
        127.0.0.1:2006=d
        127.0.0.2:2003=a
        127.0.0.2:2004=b
        127.0.0.2:2005=c
        127.0.0.2:2006=d
    ;

match ^collection.([^.]+).([^.]+).([^.]+).[^.]+.(.+)
    route using collection.\1.\2.\3.\4
    send to aggrs
    ;

match
    ^collection.([^.]+).([^.]+).([^.]+).[^.]+.(.+) (regex) -> collection.fo.bar.x.y.blakwklsa.skla
    fnv1a_ch(aggrs)
        using(collection.\1.\2.\3.\4) -> collection.fo.bar.x.blakwklsa.skla 
        127.0.0.1:2004
